toolkits/nature/packages/nature-section-heading/README.md

```diff
@@ -0,0 +1,74 @@
+# Nature Section Heading
+
+[GitHub](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/nature/packages/nature-section-heading) | [NPM](https://www.npmjs.com/package/@springernature/nature-section-heading)
+
+The Nature Section heading is a type of heading used for titles of some subsections of pages on Nature.com.
+
+It uses borders above and below the heading text to make it stand out from other headings on the page.
+
+![Example](https://i.ibb.co/28w2b0S/Screenshot-2021-12-29-at-10-44-13.png)
+
+## When to use this component
+
+Use the Nature Section heading when you want to organise multiple pieces of content into a section on a landing page. For example, a group of cards linking to articles on a similar subject.
+
+## When not to use this component
+
+Don’t use the Nature Section heading unless you’re organising multiple related items of content into a group.
+
+When you need to break up content on article pages into chunks, use [default heading styles](https://frontend-design-system.private.springernature.app/nature/styleguide/typography#headings-nature-journals).
+
+## How it works
+
+Write clear, descriptive headings using sentence case.
+
+
+### Section headings as links
+
+You may want a Section heading to link to another page.
+
+In these cases, include a chevron after the heading text to help users understand that they can click on it.
```

Comment:
Should we link to where the chevron can be found? Not sure this is the correct one but...
https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/default/img/icons/chevron-right.svg